form.blade.php

```diff
@@ -0,0 +1,204 @@
+@extends('layouts.main')
+@section('form')
+<div class="accordion container" id="accordionExample">
+    <h3>Education Details</h3>
+    {{-- 10th accordion --}}
+    <div class="">
+    <div class="accordion-item">
+      <h2 class="accordion-header" id="headingone_ssc">
+        <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#ssc_details" aria-expanded="true" aria-controls="collapseOne">
```

Comment:
aria-controls should be same as id
eg aria-controls="collapseOne" change to aria-controls="collapseOne_ssc"